modules/services/kubemonkey-values.yml.tmpl

```diff
@@ -0,0 +1,16 @@
+config:
+  dryRun: false
+  runHour: 0
+  startHour: 3
+  endHour: 5
```

Comment:
@bodymindarts I also decided to remove the variables for these parameters (adding them was premature) and stick to these defaults.